### PR TITLE
New Issue and PR Templates

### PR DESCRIPTION
**Pull request by mtarking**
New Issue and PR Templates

### CODE REVIEW

**Review by mikewiebe — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    validations:
+      required: false
+
+  - type: dropdown
+    id: role
+    attributes:
+      label: Which role is this issue related to?
+      multiple: false
+      options:
+        - validate
```
[the remark this answers is not in the document]
Comment (mtarking) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: updated all files to show fqcn

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    validations:
+      required: true
+
+  - type: dropdown
+    id: data-model
+    attributes:
+      label: Which section of the data model is this issue related to?
+      multiple: false
+      options:
+        - global
```
[the remark this answers is not in the document]
Comment (mtarking) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: prepended top-level keys with vxlan and appended defaults with vxlan

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+      required: false
+
+  - type: dropdown
+    id: role
+    attributes:
+      label: Which role is this issue related to?
+      multiple: false
+      options:
+        - cisco.nac_dc_vxlan.validate
+        - cisco.nac_dc_vxlan.create
```
Comment (mikewiebe) [inline .github/ISSUE_TEMPLATE/bug_report.yml:66]: Looks better.  One small thing...

create, deploy and remote have the additional `dtc` as part of the namespace

`cisco.nac_dc_vxlan.dtc.create`

Comment (mtarking) [reply .github/ISSUE_TEMPLATE/bug_report.yml:66]: You're right, my bust. I copied from role level part of readme quickly. Fixing and committing proper fqcn now.

**Pull request merged**